tests/api/test_workflows.py

```diff
@@ -29,11 +33,27 @@ def test_load_example_workflows_loaded(workflow_names: list[str], fastapi_client
     assert sorted(result["workflows"]) == sorted(workflow_names)
 
 
-def test_load_rules(transformation_rules, fastapi_client):
+def test_load_rules(transformation_rules, fastapi_client: TestClient):
     response = fastapi_client.get("/api/rules")
 
     # Assert
     assert response.status_code == 200
     rules = response.json()
     assert len(transformation_rules.classes) == len(rules["classes"])
     assert len(transformation_rules.properties) == len(rules["properties"])
+
+
+def test_run_default_workflow(cognite_client: CogniteClient, fastapi_client: TestClient, data_regression):
+    response = fastapi_client.post(
+        "/api/workflow/start",
+        json=RunWorkflowRequest(name="default", sync=True, config={}, start_step="Not used").dict(),
+    )
+
+    assert response.status_code == 200
+    result = response.json()["result"]
+    assert result["error_text"] is None
+    data = {}
+    for resource_name in ["assets", "relationships", "labels"]:
+        memory: MemoryClient = getattr(cognite_client, resource_name)
+        data[resource_name] = memory.dump(ordered=True, exclude={"metadata.start_time", "metadata.update_time"})
+    data_regression.check(data, basename="default_workflow")
```

Comment:
Is data_regression fixture?

Reply:
Yes, it comes from the `pytest` plugin `pytest-regression`. It simply takes the data dumps it to a a `yaml` file, and the next time the test runs checks that the data is the same as last time.

Reply:
It is thus very important that the data, which is a dictionary, is exactly the same as last time. Thus it is ordered, and the dynamic fields `start_time` and `update_time` are removed as these will change between each time the test runs.

Reply:
Aha! Now I understand! Pretty sweat !